spec/spec_helper.rb

```diff
@@ -10,6 +10,7 @@
 require 'webmock/rspec'
 require 'factory_girl'
 require 'sidekiq/testing'
+require 'shoulda/matchers'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.